src/python/pants/backend/python/subsystems/pex_build_util.py

```diff
@@ -110,6 +113,13 @@ class PexBuilderWrapper(object):
   class Factory(Subsystem):
     options_scope = 'pex-builder-wrapper'
 
+    @classmethod
+    def register_options(cls, register):
+      super(PexBuilderWrapper.Factory, cls).register_options(register)
+      register('--setuptools-version', advanced=True, default='40.6.3',
```

Comment:
This is independent from the version specified in `python_native_code.py`... should there be a subsystem to declare this? Or are they intentionally independent? and if so, should that be via a scoped subsystem dep?

It seems like this is fallout from `python-setup` no longer using this requirement to resolve these tools into the interpreter cache, but might be a good opportunity to split out `Setuptools` and `Wheel` subsystems.

Reply:
These are independent:
+ PexBuilderWrapper needs a setuptools that supports `pkg_resources.declare_namespace` for when it injects old-style namespace packages in pex loose sources.
+ python_native_code.py needs a seuptools that supports the `setup.py` of the package it is building.

A scoped subsystem may make sense for setuptools (#7576), but wheel is now only used by python_native_code.py so I'm staying lazy on that.